hypershift-operator/controllers/platform/aws/controller.go

```diff
@@ -642,3 +648,19 @@ func (r *AWSEndpointServiceReconciler) controlPlaneOperatorRoleARN(ctx context.C
 	}
 	return hc.Spec.Platform.AWS.RolesRef.ControlPlaneOperatorARN, nil
 }
+
+func (r *AWSEndpointServiceReconciler) controlPlaneOperatorRoleARNWithoutPath(ctx context.Context, hc *hyperv1.HostedCluster) (string, error) {
```

Comment:
It doesn't look like `ctx` is used

Reply:
It isn't indeed - I was kind modeling it off of https://github.com/openshift/hypershift/pull/3215/files#diff-703bbd22baf7c689e315f5c561efed7afea4f14d0ade48d66af784206a0bc421R645 

```go
func (r *AWSEndpointServiceReconciler) controlPlaneOperatorRoleARN(ctx context.Context, hc *hyperv1.HostedCluster) (string, error)
```

Which similarly has an unused context, but am happy to remove this

Reply:
Removed both contexts :D